drgn/commands/_builtin/crash/ps.py

```diff
@@ -0,0 +1,294 @@
+    CellFormat,
+    double_quote_ascii_string,
+    escape_ascii_string,
+    print_table,
+)
+from drgn.helpers.linux.kthread import task_is_kthread
+from drgn.helpers.linux.mm import get_task_rss_info, task_vsize, totalram_pages
+from drgn.helpers.linux.pid import find_task, for_each_task
+from drgn.helpers.linux.sched import (
+    task_cpu,
+    task_since_last_arrival_ns,
+    task_state_to_char,
+    thread_group_leader,
+)
+
+_NSECS_TO_SECS = 1000000000
+
+
+def get_task_arrival_time(task: Object) -> int:
+    """
+    Get a task's arrival time on cpu
+
+    A task's arrival time is only updated when the task is put ON a cpu via
+    context_switch.
+
+    :param task: ``struct task_struct *``
+    :returns: arrival time instance in ns granularity
+    """
+
+    arrival_time = task.sched_info.last_arrival.value_()
+    return arrival_time
+
+
+def format_nanosecond_duration(nanosecs: int) -> str:
+    """
+    :returns: conversion of nanoseconds to [dd hh:mm:ss.ms] format
+    """
+    secs = nanosecs / _NSECS_TO_SECS
+    dd, rem = divmod(secs, 86400)
+    hh, rem = divmod(rem, 3600)
+    mm, secs = divmod(rem, 60)
+    return "%02ld %02ld:%02ld:%06.3f" % (dd, hh, mm, secs)
+
+
+def show_tasks_last_runtime(tasks: List[Object]) -> None:
+    """
+    Display task information in their last arrival order.
+    """
+    tasks.sort(key=task_since_last_arrival_ns)
+    for t in tasks:
+        cpu = task_cpu(t)
+        pid = t.pid.value_()
+        state = task_state_to_char(t)
+        command = double_quote_ascii_string(t.comm.string_())
+        time_nanosec = task_since_last_arrival_ns(t)
+        last_arrival = format_nanosecond_duration(time_nanosec)
+        print(
+            f"[{last_arrival:>14}] [{state}]  PID: {pid:<8} TASK: {t.value_():x}   CPU: {cpu:<2}  COMMAND: {command}"
+        )
+
+
+def show_tasks_last_runtime_timestamp(tasks: List[Object]) -> None:
+    """
+    Display task information sorted by last_run/timestamp/last_arrival, showing the raw timestamp.
+    """
+    tasks.sort(key=get_task_arrival_time, reverse=True)
+    for t in tasks:
+        cpu = task_cpu(t)
+        pid = t.pid.value_()
+        state = task_state_to_char(t)
+        command = double_quote_ascii_string(t.comm.string_())
+        last_arrival = get_task_arrival_time(t)
+        print(
+            f"[{last_arrival:>14}] [{state}]  PID: {pid:<8} TASK: {t.value_():x}   CPU: {cpu:<2}  COMMAND: {command}"
+        )
+
+
+def show_taskinfo(prog: Program, tasks: List[Object]) -> None:
+    """
+    Display tasks informations.
+    """
+    rows: List[Sequence[Any]] = [
+        (
+            CellFormat("PID", ">"),
+            CellFormat("PPID", ">"),
+            CellFormat("CPU", "^"),
+            CellFormat("TASK", "^"),
+            CellFormat("ST", ">"),
+            CellFormat("%MEM", ">"),
+            CellFormat("%VSZ", ">"),
+            CellFormat("RSS", ">"),
+            CellFormat("COMM", "<"),
```

Comment:
"<" is the default for strings, so this one is unnecessary to wrap

```suggestion
            "COMM",
```